src/smpp_gateway/management/commands/smpp_client.py

```diff
@@ -36,7 +41,7 @@ def add_arguments(self, parser):
         )
         parser.add_argument(
             "--submit-sm-params",
-            default=os.environ.get("SMPPLIB_SUBMIT_SM_PARAMS"),
+            default=os.environ.get("SMPPLIB_SUBMIT_SM_PARAMS", r"{}"),
```

Comment:
This gets `json.loads()`ed then joined with another dict, so this should default to something that plays well with that.